src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp

```diff
@@ -711,15 +711,15 @@ void SolidMechanicsLagrangianFEM::applyDisplacementBCImplicit( real64 const time
         "\nWarning!"
         "\n{} {}: There is no displacement boundary condition applied to this problem in the {} direction. \n"
         "The problem may be ill-posed.\n";
-      GEOS_WARNING_IF( isDisplacementBCAppliedGlobal[0] == 0, // target set is empty
-                       GEOS_FMT( bcLogMessage,
-                                 getCatalogName(), getDataContext(), 'x' ) );
-      GEOS_WARNING_IF( isDisplacementBCAppliedGlobal[1] == 0, // target set is empty
-                       GEOS_FMT( bcLogMessage,
-                                 getCatalogName(), getDataContext(), 'y' ) );
-      GEOS_WARNING_IF( isDisplacementBCAppliedGlobal[2] == 0, // target set is empty
-                       GEOS_FMT( bcLogMessage,
-                                 getCatalogName(), getDataContext(), 'z' ) );
+      GEOS_WARNING_CTX_IF( isDisplacementBCAppliedGlobal[0] == 0, // target set is empty
+                           GEOS_FMT( bcLogMessage,
+                                     getCatalogName(), getDataContext(), 'x' ), getDataContext() );
```

Comment:
I think in cases such as this where the context was already in the error message, we should remove it. I'd be curious to see how the extra (redundant) context shows up in the message.